Server/Game/Commands/Misc/HelpCommand.cs

```diff
@@ -0,0 +1,45 @@
+using Platform_Racing_3_Server.Game.Client;
+using Platform_Racing_3_Server_API.Game.Commands;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Platform_Racing_3_Server.Game.Commands.Misc
+{
+    internal class HelpCommand : ICommand
+    {
+        public string Permission => null;
+
+        public void OnCommand(ICommandExecutor executor, string label, ReadOnlySpan<string> args)
+        {
+            if (executor is ClientSession session)
+            {
+                executor.SendMessage("<b><u>Developer / Administrator / Manager</u></b>"
```

Comment:
This is pretty terrible way to implement this, going through all available commands and checking for HasPermission is the proper way. Also adding Description getter to ICommand is better way to implement this.

Reply:
Yup, that's what I was talking about at the top. Just made it this way temporarily until I make the proper one. I'll let you know once it's done. :smiley: